sklearn/grid_search.py

```diff
@@ -605,9 +605,13 @@ class GridSearchCV(BaseSearchCV):
 
     Important members are fit, predict.
```

Comment:
Now the other members are unhappy with the "importance" given to `fit` and `predict`